crates/core/src/db/queries/metadatas.rs

```diff
@@ -291,8 +294,11 @@ pub fn list(
                 Expr::tbl(Metadatas::Table, Metadatas::Address)
                     .equals(MetadataCreators::Table, MetadataCreators::MetadataAddress),
             )
-            .and_where(Expr::col(MetadataCreators::CreatorAddress).is_in(creators))
-            .and_where(Expr::col(MetadataCreators::Verified).eq(true));
+            .and_where(Expr::col(MetadataCreators::CreatorAddress).is_in(creators));
+    }
+
+    if !matches!(allow_unverified, Some(true)) {
+        query.and_where(Expr::col(MetadataCreators::Verified).eq(true));
     }
```

Comment:
```suggestion
    if allow_unverified != Some(true) {
        query.and_where(Expr::col(MetadataCreators::Verified).eq(true));
    }
```

Does this not work. reads better.